tweag/nixpkgs-graph
# Combine all the installation steps into the nix shell

## What this PR does
This merges all the installation steps into `shell.nix`.

### Principal changes:
We deleted the `build.sh` file and ported the install steps to `shell.nix` file. Now all default commands can be executed via nix-shell.

### Compared with the version \#23
- This removed the build.sh file.
- This merged all the installation steps into `shell.nix`.
- This lets `nixpkgs_graph.py` file output a special `start.graphml` file to start Neo4j.

## How to test this PR
- Install nix package manager if this is not done:
  ```sh
  $ curl -L https://nixos.org/nix/install | sh
  ``` 

- To test the CLI, run nix shell:
  ```sh
  nix-shell
  ```
  And you should have the following result
  ```
  +-----------------+
  | number_of_nodes |
  +-----------------+
  | 64205           |
  +-----------------+
  ```
